controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java

```diff
@@ -71,6 +73,24 @@ public void maintain() {
         reportRemainingRotations();
         reportQueuedNameServiceRequests();
         reportInfrastructureUpgradeMetrics();
+        reportAuditLog();
+    }
+
+    private void reportAuditLog() {
+        final String OPERATOR = "operator";
+
+        AuditLog log = controller().auditLogger().readLog();
+        for (AuditLog.Entry entry : log.entries()) {
+            String[] resource = entry.resource().split("/");
+            String operationMetric;
+            if(resource[1] != null) {
+                String api = resource[1];
+                operationMetric = OPERATION_PREFIX + api;
+                Metric.Context context = metric.createContext(Map.of(OPERATOR, entry.principal()));
+
+                metric.add(operationMetric, 1, context);
```

Comment:
Since `MetricsReporter` runs at an interval and this loops over all entries every time, this metric will end up counting the same entries multiple times.

I suggest that you instead calculate the total count and use `metric.set`. You can verify that the metric is as expected by calling `metricsReporter.maintain()` more than once in the test.